src/browser/rvm/rvm_message_bus.ts

```diff
@@ -147,15 +147,28 @@ export interface ApplicationEvent extends RvmMsgBase {
     sourceUrl: string;
 }
 
+export interface LicenseInfo {
+    licenseKey?: string;
+    client?: {
+        type: 'dotnet' | 'java' | 'air' | 'node' | 'js';
```

Comment:
For `js` it will be internal. This is also servers as a point of truth or if / when validation is added